src/App.test.js

```diff
@@ -13,10 +13,18 @@ import Enzyme, { shallow } from 'enzyme';
 import EnzymeAdapter from 'enzyme-adapter-react-16';
 
 import App from './App';
+
 Enzyme.configure({ adapter: new EnzymeAdapter() })
-it('renders without crashing', () => {
- const wrapper = shallow(<App />);
- const appComponent = wrapper.find("[data-test='component-app']");
+
+const setup = (props={}, setup=null) => {
+    return shallow(<App { ...props} />)
+}
+const findByTestAttr = (wrapper, val) => {
+    return wrapper.find(`[data-test="${val}"]`);
+}
+test('renders without crashing', () => {
```

Comment:
You have total 1 test cases for whole project.

Reply:
No sir I write test cases on NavigatioItem.test , App.test, BurgerBuilder.test, auth.test, logo.test,BuildControl.test, spinner.test